packages/contracts/test/test-utils/uups-upgradeable.ts

```diff
@@ -1,8 +1,12 @@
 import {SignerWithAddress} from '@nomiclabs/hardhat-ethers/signers';
 import {expect} from 'chai';
 import {Contract} from 'ethers';
+import {defaultAbiCoder} from 'ethers/lib/utils';
 import {ethers} from 'hardhat';
 
+export const IMPLEMENTATION_SLOT =
```

Comment:
How is this value computed? Is this a well-known entry point for the UUPS proxy?

Reply:
It is from the EIP, I will add the link as a comment. https://eips.ethereum.org/EIPS/eip-1967

Reply:
Clarified in f437dbc32e0f0c6b816d82295c4a353385ac2b19